cmd/nerdctl/builder/builder.go

```diff
@@ -146,6 +147,7 @@ func debugCommand() *cobra.Command {
 	cmd.Flags().String("image", "", "Image to use for debugging stage")
 	cmd.Flags().StringArray("ssh", nil, "Allow forwarding SSH agent to the build. Format: default|<id>[=<socket>|<key>[,<key>]]")
 	cmd.Flags().StringArray("secret", nil, "Expose secret value to the build. Format: id=secretname,src=filepath")
+	helpers.AddDurationFlag(cmd, "buildg-startup-timeout", nil, 1*time.Minute, "", "Timeout for starting up buildg")
```

Comment:
so the unit here is `Nanosecond`? I feel it will be hard to let users calculate the nano sec number and put it here :)

maybe use an explicit unit here (e.g., Millisecond or Second) and we do the conversion (or at least clarify the unit `nano second` in explanation.)

Reply:
No, time.ParseDuration-supported values can be used here (e.g. `1m`, `3s`, etc.).

c.f. *Command line flag syntax* https://pkg.go.dev/github.com/spf13/pflag

> Duration flags accept any input valid for time.ParseDuration.